internal/cmd/group_tokens.go

```diff
@@ -120,3 +130,18 @@ var groupCreateTokenCmd = &cobra.Command{
 		return nil
 	},
 }
+
+func validateDBNames(dbNames []string) ([]string, error) {
```

Comment:
I suggest changing the API to take database names instead, since it is how we do in most (if not all) cases.